lib/logger.js

```diff
@@ -52,6 +52,14 @@ export default class Logger {
     this.showMessages(this._label, filteredMessages)
   }
 
+  static getMaxType (messages) {
```

Comment:
I agree. I am almost tempted to do a rename `type` in all the message objects. `type` is  such a generic label. Maybe rename to `severity`. What do you think?